02.extensions/zoolingual/01.basics/03.integration/docs.md

```diff
@@ -4,18 +4,45 @@ taxonomy:
     category: docs
 ---
 
-Once **ZOOlingual** is installed all ZOO elements will have a new section available in the configuration for language managing. It allows to set on which language/s that specific element will be displayed, falling back to all if none selected.
+Once **ZOOlingual** is installed all ZOO elements will have a new section available in the configuration for language managing:
 
-The language can be set on the element **Configuration** and/or **Assignment** allowing to set a base restriction in the Configuration and override it in the layout positions assignment. This powerful feature is specially handy when dealing with number/date values elements, like a Birthday, which value requires no translation, but rather display localization; assigning the element on the same position once for each language and adjusting its display options for it, will do the trick!
+![ZOOlingual section in Element configuration](section-for-element.png)
+
+It allows to set on which language/s that specific element will be displayed. If no particular language is selected, the element will be displayed for all languages.
+
+The language can be set in two ways:
+
+* on the element **Configuration** - allows to set a base restriction
+* on the layout positions **Assignment** - allows to override the language from the configuration.
+
+For example, this powerful feature is specially handy when dealing with number/date values elements, like a Birthday, which value requires no translation, but rather display localization:
+
+![Assignment layout - instances](example-assignment-layout.png)
+
+Assigning the element on the same position once for each language and adjusting its display options for it, will do the trick!
+
+![Assignment layout - instances](example-several-instances.png)
 
 >>> Tag translation and translation through Submission is currently unsupported.
 
 ## Content
 
+On the **Item Edit** page the element value for each language could be filled in:
+
+![Assignment layout - instances](item-edit-page.png)
+
 For those elements that do require a translation we would duplicate it for each language and set that specific language in the Element Configuration. That way, that element becomes language specific and we can start inputing the translated content right away while editing the Items as usual.
```

Comment:
Update this text, instead of "we would duplicate" write it as "you would..." to make it more clear that this part is no automatic.